docs/source/basic_usage.rst

```diff
@@ -33,87 +33,153 @@ Interpreting a Device
 of keeping your Python API and display in sync, making the transition from
 using screens to using an IPython shell seamless.
 
-Take for instance the built-in ``ophyd.EpicsMotor`` class, we can do a quick
-survey of what the default ``read`` and ``configuration`` attributes are:
+For the following applications we'll use the ``motor`` simulation contained
+within ``ophyd`` itself. We also need to create a ``QApplication`` before we
+create any widgets:
 
 .. ipython:: python
 
-    import ophyd
+   from qtpy.QtWidgets import QApplication
+   app = QApplication([])
 
-    ophyd.EpicsMotor._default_read_attrs
 
-    ophyd.EpicsMotor._default_configuration_attrs
+.. ipython:: python
+    :suppress:
+    
+    from ophyd.sim import motor
 
 
-These ``read`` and ``configuration`` attributes can always be changed when the
-device is instantiated.  However, this is only a small subset of the signals
-that are associated with the class.
 
-.. ipython:: python
+Using Happi
+^^^^^^^^^^^
+``happi`` is not a requirement for using ``typhon``. However, it is certainly
+recommended. For more information, visit the `GitHub
+<https://github.com/pcdshub/happi/>`_ repository. The main purpose of the
+package is to store information on our Ophyd objects so that we can load them
+in a variety of contexts. If you do not use ``happi`` you will need to create
+your objects and displays in the same process.
 
-   ophyd.EpicsMotor._sig_attrs
+Here is a quick example if you wanted to get a feel for what ``typhon`` looks
+like with `happi``
 
-Now, at first glance it may not be obvious, but there is a lot of information
-here! We have a Python abstraction capable of instantiating a number of useful
-EPICS signals based on a given ``prefix``. We know which of these signals an
-operator will want to control and which ones are purely meant to be read back.
-We also have these signals grouped by their importance to operation, each with
-a terse human legible description of what the PV represents.
-
-Typhon takes advantage of this to generate a concise PyDM user display. The
-:class:`.TyphonDeviceDisplay` uses the signal groups; ``read_attrs``,
-``configuration_attrs`` and ``hints`` to generate plots and widgets based on
-the type and class of EPICSSignals. In order to best select the widget types,
-:class:`.TyphonDeviceDisplay` attempts to connect to all of the PVs listed. If this
-is not possible, there are ways to manually configure which widget will be
-used. Simply invoke your device and then create your ``PyDMApplication`` and
-widget
+.. code:: python
+
+    import happi
+    from typhon.plugins import register_client
+
+    # Initialize a new JSON based client
+    client = happi.Client(path='db.json', initialize=True)
+    # Register this with typhon
+    register_client(client)
+    # Add a device to our new database
+    device = happi.Device(device_class='ophyd.sim.SynAxis',
+                          prefix='Tst:Mtr', args=[], kwargs={{name}},
```

Comment:
`kwargs={{name}}` ?

Reply:
It is a Jinja template and should be a string